cookbooks/travis_python/recipes/pyenv.rb

```diff
@@ -155,7 +167,7 @@
   end
 
   execute "install packages #{py}" do
-    command "#{virtualenv_root}/#{pyname}/bin/pip install --upgrade #{packages.join(' ')}"
+    command "#{virtualenv_root}/#{pyname}/bin/pip install -I -U #{packages.join(' ')}"
```

Comment:
I suppose I was concerned about system-level stuff muddying the waters.  I'll gladly back it out :+1: